CHANGELOG.md

```diff
@@ -15,7 +15,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Bumps `base64` from ^0.13 to ^0.20 ([#95](https://github.com/opensearch-project/opensearch-rs/pull/95), [#105](https://github.com/opensearch-project/opensearch-rs/pull/105))
 - Bumps `aws-*` from >=0.10 to >=0.53 ([#108](https://github.com/opensearch-project/opensearch-rs/pull/108))
 - Bumps `toml` from 0.5.6 to 0.7.1
-- Bumps `sysinfo` from 0.26.4 to 0.27.7
```

Comment:
Looks like changelog needs some help ...

Reply:
It's strange because I'm trying to reproduce it as a test case in the dependabot-changelog-helper, and copy pasting the changelog verbatim it's still updating correctly in the test case :/

Will continue trying to reproduce and get a fix for it

Reply:
Thanks @Xtansia , there was another occurrence in core recently [1], it it could be helpful

[1] https://github.com/opensearch-project/OpenSearch/pull/6379/files

Reply:
Have found the issue and PRd a fix to the action https://github.com/dangoslen/dependabot-changelog-helper/pull/132